test/fixtures/google-gax-packaging-test-app/src/index.js

```diff
@@ -161,7 +162,7 @@ async function testEchoError(client) {
       await client.echo(request);
   } catch (err) {
       clearTimeout(timer);
-      assert.strictEqual(JSON.stringify(err.statusDetails), JSON.stringify(expectedDetails));
+      assert.strictEqual(rest ? JSON.stringify(err.details) : JSON.stringify(err.statusDetails), JSON.stringify(expectedDetails));
```

Comment:
I think we should just make sure this field on error is the same for both DIREGAPIC and GAPIC>

Reply:
Thanks for the advice. I update the name.